src/lib/services/chat.ts

```
@@ -1,58 +1,172 @@
+			try {
+				const responseData = typeof error.responseBody === 'string'
+					? JSON.parse(error.responseBody)
+					: error.responseBody;
+
+				if (responseData?.error) {
+					console.error('  OpenAI Error Code:', responseData.error.code);
+					console.error('  OpenAI Error Type:', responseData.error.type);
+					console.error('  OpenAI Error Message:', responseData.error.message);
+					console.error('  OpenAI Error Param:', responseData.error.param);
+				}
+			} catch (parseError) {
```

ESLint

[error] 'parseError' is defined but never used.

(@typescript-eslint/no-unused-vars)